doc/sphinx-guides/source/container/index.rst

```diff
@@ -0,0 +1,26 @@
+Container Guide
+===============
+
+**Contents:**
+
+.. toctree::
+
+  base-image
+
+Running Dataverse software in containers is quite different than in a :doc:`classic installation <../installation/prep>`.
+
+Both approaches have pros and cons. These days (2022) containers are very often used for development and testing,
```

Comment:
Remove (2022), unless you're sure that you'll remember to edit it in later years (more hassle). I agree that containers are becoming ever more present and I don't see that changing very soon.

Reply:
```suggestion
Both approaches have pros and cons. These days containers are very often used for development and testing,
```